src/playwrightTest.ts

```diff
@@ -71,12 +71,11 @@ export class PlaywrightTest {
         return { cli: '', version: v };
 
       // Resolve playwright-core relative to @playwright/test.
-      const coreInfo = await this._runNode([
+      const cliInfo = await this._runNode([
         '-e',
-        'try { const coreIndex = require.resolve("playwright-core"); console.log(JSON.stringify({ coreIndex })); } catch { console.log("undefined"); }',
-      ], path.dirname(pwtIndex));
-      const { coreIndex } = JSON.parse(coreInfo);
-      let cli = path.resolve(coreIndex, '..', 'lib', 'cli', 'cli');
+        'try { const cli = require.resolve("@playwright/test/cli"); console.log(JSON.stringify({ cli })); } catch { console.log("undefined"); }',
+      ], path.dirname(configFilePath), settingsEnv);
+      let { cli } = JSON.parse(cliInfo);
 
       // Dogfood for 'ttest'
       if (cli.includes('packages/playwright-core') && configFilePath.includes('playwright-test'))
```

Comment:
this dogfood seems broken after this change. We used it when we are inside our repository (microsoft/playwright). 

```suggestion
      if (cli.includes('packages/playwright-test') && configFilePath.includes('playwright-test'))
```
it should work after that.